src/component_builder/config.py

```diff
@@ -1,5 +1,8 @@
 import os.path
-from ConfigParser import ConfigParser
+try:
+    from configparser import ConfigParser
+except ImportError:
+    from ConfigParser import ConfigParser
```

Comment:
Re: python2 vs python3. I wonder what the best way would be to ensure this in tests.

Probably `make test` *does* need to care about virtualenvs after all.